.gitpod.yml

```diff
@@ -0,0 +1,11 @@
+image:
+  file: .gitpod.Dockerfile
+tasks:
+  - init: make dev-gitpod
+vscode:
+  extensions:
+    - ms-python.python
+    - littlefoxteam.vscode-python-test-adapter
+    - ms-azuretools.vscode-docker
+    - davidanson.vscode-markdownlint
+    - bungcip.better-toml
```

Comment:
Can you configure your editor to add a new line at the end of a file? This avoids unnecessary diffs in the future

https://thoughtbot.com/blog/no-newline-at-end-of-file#following-the-rules-in-your-editor

Reply:
Hi Rubens, added!